scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1338,6 +1338,10 @@ def start_service(self, service, idx):
     fid = fetch_fid(self, service, idx)
     if fid == -1:
         return -1
+    cmd = "/opt/seagate/cortx/motr/libexec/m0trace_logrotate.sh"
```

Comment:
Add a comment, that it needs to be executed only after copying /etc/sysconfig/motr file.